### PR TITLE
Kastparser

### PR DESCRIPTION
**Pull request by liyili2**
first version.

### CODE REVIEW

File: bin/kompile [hunk 1]
```diff
@@ -4,7 +4,11 @@ export PATH=$PATH:"$(dirname "$0")/../lib/native/cygwin":"$(dirname "$0")/../lib
 export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$(dirname "$0")/../lib/native/linux":"$(dirname "$0")/../lib/native/linux/x64"
 ulimit -s `ulimit -H -s`
 if [ -z "$K_OPTS" ];
+<<<<<<< HEAD
+  then export K_OPTS="-Xms64m -Xmx2048m -Xss64m"
+=======
   then export K_OPTS="-Xms64m -Xmx1024m -Xss32m -XX:+TieredCompilation"
+>>>>>>> unified-builtins2
```
Comment (radumereuta) [inline bin/kompile]: Is this a conflict, or what?
This pull request shouldn't modify this file.

Comment (liyili2) [inline bin/kompile]: It is my mistake when I created the branch. At that time, I needed to set up 2048 m memory for my C semantics. Now, I just change it back.

**Comment by liyili2** [comment]
@dwightguth, Are you ok with the mess in the pull request, or do you want me to use the new unified-builtins2 branch, put all my changes in one commit and re-submit the pull request again?

**Comment by radumereuta** [comment]
The pull request looks fine this time. We should wait for jenkins to start testing.
I should have started already.

**Comment by radumereuta** [comment]
Jenkins: ok to test

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by dwightguth** [comment]
Radu: you're not an admin on the k framework, you can't approve pull requests for testing.

**Comment by radumereuta** [comment]
Is there a list some where with who can do what?

**Comment by liyili2** [comment]
@grosu and @dwightguth, please review.

**Comment by dwightguth** [comment]
Jenkins: ok to test

**Comment by dwightguth** [comment]
Before I review this I'd like you to rebase out your changes to the files that are not part of this diff. basically remove the changes to the files that aren't actually needed and then squash what's left into a single commit to lose the history of changes to unnecessary files.

**Pull request closed without merging**